src/mococrw/openssl_wrap.h

```diff
@@ -45,10 +45,20 @@ enum class DigestTypes {
     NONE = std::numeric_limits<int>::max()
 };
 
+/**
+ * Enum for the cipher types that can be used for CMAC computations.
+ */
+enum class CmacCipherTypes {
+    AES_CBC_128,
+    AES_CBC_256,
+    NONE = std::numeric_limits<int>::max()
```

Comment:
I know that should have happened already, but can you please `#include <limits>` for `std::numeric_limits`?

Reply:
Yep.
On the other hand nothing uses this at all. Do you mind just removing `NONE` and, if your linter on CI is good enough, perhaps also remove the `default` case in the mapper `_getCipherPtrFromCmacCipherType()`?
(I have just used that construct because it was done that way for HMAC.)

Reply:
The problem with doing that is that it would be inconsistent to not have it for CmacCipherTypes but keep it for HMAC, but removing it from HMAC is an ABI and API change. Let's just keep it.

Reply:
Done.

Reply:
Honestly, I kind of agree that not having NONE is better design. I fear we have this implict optional build-in at other places for digests but I don't think that its a good idea to continue this. We should rather reconsider the NONE on DigestTypes in the future…

Reply:
Removed `None` from `CmacCipherTypes`.